Packs/QRadar/Integrations/QRadar_v3/README.md

```diff
@@ -51,6 +51,8 @@ The *Query to fetch offenses* feature enables defining a specific query for offe
 If you change the *Query to fetch offenses* value, it will not re-fetch offenses that have already been fetched. To re-fetch those offenses, run the ***qradar-reset-last-run*** command.  
 **Note:**  
 The list of QRadar IDs that were already fetched will be reset and duplicate offenses could be re-fetched, depending on the user query.  
+
+In case of setting a query with the intent to fetch specific offenses or older offenses, make sure that the *First fetch timestamp* parameter is set to a value that covers the desired time range for the offenses to be fetched, if using an already existing instance that has performed fetch, run the ***qradar-reset-last-run*** command to ensure before starting the fetch with the new query.
```

Comment:
```suggestion
In case of setting a query with the intent to fetch specific offenses or older offenses, make sure that the *First fetch timestamp* parameter is set to a value that covers the desired time range for the offenses to be fetched, if using an already existing instance that has performed fetch, run the ***qradar-reset-last-run*** command to before starting the fetch with the new query.
```